ui/config-overrides.js

```diff
@@ -17,16 +17,31 @@ const WebpackBar = require('webpackbar')
 const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 const rewireHtmlWebpackPlugin = require('react-app-rewire-html-webpack-plugin')
 
+function copyDistroRes() {
+  const distroResPath = '../bin/distro-res'
+  if (fs.existsSync(distroResPath)) {
+    fs.copySync(distroResPath, './public/distro-res')
+  }
+}
+
 function injectDistroToHTML(config, env) {
```

Comment:
Deserves some comments describing the behavior, since it is different in dev mode and production mode.